runtime/protocol/http-client-engines/http-client-engine-crt/common/test/aws/smithy/kotlin/runtime/http/engine/crt/RequestConversionTest.kt

```diff
@@ -91,20 +91,6 @@ class RequestConversionTest {
         crtBody.cancel()
     }
 
-    @Test
-    fun testEngineDoesNotAddContentLengthHeaderForEmptyBody() {
-        val request = HttpRequest(
-            HttpMethod.POST,
-            Url.parse("https://test.aws.com?foo=bar"),
-            Headers.Empty,
-            HttpBody.Empty,
-        )
-
-        val testContext = EmptyCoroutineContext + Job()
-        val crtRequest = request.toCrtRequest(testContext)
-        assertFalse(crtRequest.headers.contains("Content-Length"))
-    }
-
```

Comment:
**Suggestion**: I recommend keeping this test but inverting the expectation and the name.